org/ystia/topologies/a4c_yorc_basic/types.yml

```diff
@@ -0,0 +1,273 @@
+tosca_definitions_version: alien_dsl_2_0_0
+
+metadata:
+  template_name: org.ystia.topologies.a4c_yorc_basic
+  template_version: 2.1.0-SNAPSHOT
+  template_author: Ystia
+
+description: ""
+
+imports:
+- tosca-normative-types:1.0.0-ALIEN20
+
+- org.alien4cloud.alien4cloud.pub:2.0.0
+- org.alien4cloud.alien4cloud.webapp:2.0.0
+- org.alien4cloud.java.jdk.linux:2.0.0
+
+- org.ystia.terraform.linux.terraform:2.1.0-SNAPSHOT
+- org.ystia.yorc.alien4cloud:2.1.0-SNAPSHOT
+- org.ystia.ansible.linux.ansible:2.1.0-SNAPSHOT
+- org.ystia.terraform.pub:2.1.0-SNAPSHOT
+- org.ystia.yorc.linux.ansible:2.1.0-SNAPSHOT
+- org.ystia.yorc.experimental.consul.linux.ansible:2.1.0-SNAPSHOT
+- org.ystia.yorc.pub:2.1.0-SNAPSHOT
+- org.ystia.ansible.pub:2.1.0-SNAPSHOT
+- org.ystia.yorc.experimental.consul.pub:2.1.0-SNAPSHOT
+
+
+repositories:
+  Pulp:
+    url: http://10.197.135.244/starlings/alien4cloud
+    type: http
+
+topology_template:
+  node_templates:
+    OracleJDK:
+      type: org.alien4cloud.java.jdk.linux.nodes.OracleJDK
+      properties:
+        java_url: "https://edelivery.oracle.com/otn-pub/java/jdk/8u131-b11/d54c1d3a095b4ff2b6607d096fa80163/jdk-8u131-linux-x64.tar.gz"
+        java_home: "/opt/java"
+        component_version: "1.8.0-131-b11"
+      requirements:
+      - hostedOnComputeHost:
+          type_requirement: host
+          node: A4C_Yorc
+          capability: tosca.capabilities.Container
+          relationship: tosca.relationships.HostedOn
+    Network:
+      metadata:
+        a4c_edit_x: 179
+        a4c_edit_y: "-132"
+      type: tosca.nodes.Network
+      properties:
+        ip_version: 4
+    Alien4Cloud:
+      type: org.alien4cloud.alien4cloud.webapp.nodes.Alien4Cloud
+      properties:
+        context_root: "/"
+        component_version: "2.0.0"
+        data_dir: "/opt/alien4cloud/data"
+      requirements:
+      - javaSoftwareHostedOnJdkOracleJdkJdk:
+          type_requirement: java
+          node: OracleJDK
+          capability: org.alien4cloud.java.pub.capabilities.JDK
+          relationship: org.alien4cloud.java.pub.relationships.JavaSoftwareHostedOnJDK
+      capabilities:
+        consul:
+          properties:
+            tls_enabled: false
+            protocol: tcp
+            secure: false
+            network_name: PRIVATE
+            initiator: source
+        jolokia:
+          properties:
+            port: 8778
+        http:
+          properties:
+            port: 9200
+            protocol: http
+            secure: false
+            network_name: PRIVATE
+            initiator: source
+        rest:
+          properties:
+            user: admin
+            password: admin
+            port: 8080
+            protocol: http
+            secure: false
+            network_name: PRIVATE
+            initiator: source
+      artifacts:
+        alien_dist:
+          file: alien4cloud-premium-dist-2.0.0-dist.tar.gz
```

Comment:
please use something usable for everyone with an opensource version of alien

Reply:
sure